src/v2/Apps/Components/HorizontalPadding.tsx

```diff
@@ -1,36 +1,14 @@
-import { Box, media, space } from "@artsy/palette"
-import styled, { css } from "styled-components"
-import { SpaceProps } from "styled-system"
+import { Box, BoxProps } from "@artsy/palette"
+import styled from "styled-components"
 
-export interface HorizontalPaddingProps {
-  px?: SpaceProps["px"]
-}
+export interface HorizontalPaddingProps extends BoxProps {}
 
-export const HorizontalPadding = styled(Box)<HorizontalPaddingProps>`
-  ${p =>
-    media.xs`
-      padding-right: ${
-        // @ts-expect-error STRICT_NULL_CHECK
-        (p.px[0] && space(p.px[0])) || 0
-      }px;
-      padding-left: ${
-        // @ts-expect-error STRICT_NULL_CHECK
-        (p.px[0] && space(p.px[0])) || 0
-      }px;
-    `};
-  ${p =>
-    // @ts-expect-error STRICT_NULL_CHECK
-    p.px[1] &&
-    css`
-      padding-right: ${// @ts-expect-error STRICT_NULL_CHECK
-      space(p.px[1])}px;
-      padding-left: ${// @ts-expect-error STRICT_NULL_CHECK
-      space(p.px[1])}px;
-    `};
```

Comment:
All this was doing was just manually allowing padding overwrites — which doesn't need to be done if you use `defaultProps`

Reply:
heh, nice refactor